filters/user-defined-record-filter/spec.yml

```diff
@@ -0,0 +1,15 @@
+---
+kind       : Record
+name       : User-defined filter
+key        : user-defined-record-filter
+config     :
+-   name: code
+    label: Code
+    type: text
+description: Custom user-defined filter defined in Python
```

Comment:
This description matches `filters/user-defined-filter/spec.yml` exactly. Please make sure to distinguish consistently between record-level and field-level operations.

Reply:
Done: d29a1aa98adeb3be0e6ed567b1d4a7f22d4033f9

Reply:
Thanks! :)